src/app/components/Header/index.jsx

```diff
@@ -1,16 +1,26 @@
 import React from 'react';
 import styled from 'styled-components';
-import { C_POSTBOX, GEL_SPACING_DBL } from '../../lib/constants/styles';
+import {
+  C_POSTBOX,
```

Comment:
A bit of a nitpick, but alphabetically listing these can be good.

Reply:
Going to implement this on https://github.com/bbc/simorgh/pull/387/files as it is further ahead than this PR.